src/Store/ChromaDB/Store.php

```diff
@@ -55,10 +55,11 @@ public function query(Vector $vector, array $options = []): array
 
         $documents = [];
         for ($i = 0; $i < count($queryResponse->metadatas[0]); ++$i) {
-            $documents[] = Document::fromVector(
-                new Vector($queryResponse->embeddings[0][$i]),
-                Uuid::fromString($queryResponse->ids[0][$i]),
-                new Metadata($queryResponse->metadatas[0][$i]),
+            $documents[] = new Document( // not sure how to insert and get the text here @chr-hertel
+                id: Uuid::fromString($queryResponse->ids[0][$i]),
+                text: '???',
```

Comment:
- [ ] How could we add the text to Chroma?

cc @chr-hertel

Reply:
i guess we would store the text in chroma. not doing it atm at least.